docs/README.md

```diff
@@ -0,0 +1,23 @@
+## Generating documentation with Sphinx
+
+The best tutorial for generating documentation can be found [here](http://gisellezeno.com/tutorials/sphinx-for-python-documentation.html)
+
+Files in the source directory are generally static and should not need to be updated.  If another python module is created in `../rosette`, then the source may need to be regenerated using
+
+`sphinx-apidoc -f -o source/ ../rosette/`
```

Comment:
That's a good question, which I tried to answer with the preceding statement.  If additional modules are added to the source directory, then the sphinx-apidoc command would need to be run to create the additional rst files.  Otherwise, running it is unnecessary.  I'm open to better wording.